client/wc-api/wp-data-store/create-api-client.js

```diff
@@ -0,0 +1,52 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { ApiClient } from '@fresh-data/framework';
+import { createStore as createReduxStore } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import reducer from './reducer';
+
+function createStore( name ) {
+	const devTools = window.__REDUX_DEVTOOLS_EXTENSION__;
+
+	return createReduxStore( reducer, devTools && devTools( { name: name, instanceId: name } ) );
+}
+
+function createDataHandlers( store ) {
+	return {
+		dataRequested: resourceNames => {
+			store.dispatch( {
+				type: 'FRESH_DATA_REQUESTED',
+				resourceNames,
+				time: new Date(),
```

Comment:
I think the right answer here is not to send in "UTC" dates because all dates are UTC dates because they're based on milliseconds in UTC. (try it out, make a date in the browser console with `const now = new Date()` and print it out, change your timezone, then print it out again).

Just to note, since the timezone info is saved in the timestamp string, it should work correctly when comparing them as date objects again. However, I agree they should be saved as UTC strings anyway.

I think the right way to fix this is in `fresh-data`, where dates are serialized into strings. I created an issue to track that: https://github.com/Automattic/fresh-data/issues/114